parm/config/gfs/config.fcst

```diff
@@ -282,19 +282,11 @@ if [[ "${CDUMP}" =~ "gdas" ]] ; then # GDAS cycle specific parameters
     # Variables used in DA cycling
     export DIAG_TABLE="${HOMEgfs}/parm/parm_fv3diag/diag_table_da"
 
-    # Write restart files, where $number is current model start time.
-    # restart_interval:        $number
-    #    number=0, writes out restart files at the end of forecast.
-    #    number>0, writes out restart files at the frequency of $number and at the end of forecast.
-    # restart_interval:        "$number -1"
-    #    writes out restart files only once at $number forecast hour.
-    # restart_interval:        "$number1 $number2 $number3 ..."
-    #    writes out restart file at the specified forecast hours
-    export restart_interval=${restart_interval:-6}
-
-    # For IAU, write restarts at beginning of window also
+
     if [[ "${DOIAU}" == "YES" ]]; then
-       export restart_interval="3 6"
+      export restart_interval="3"
+    else
```

Comment:
I thought the IAU only rewinded `(T - 0.5*DA window)`? Further, why this change? This is confusing.

Reply:
`restart_interval = 3` will write restarts **every** 3 hours.
`restart_interval = "3 6"` will write restarts **at** hours `3` and `6` only.

This change is necessary to address the issue this PR is resolving.

Reply:
Ah, OK got it. Sorry, this wasn't obvious to me. Thank you!